tests/run_wgrib2_tests.sh

```diff
@@ -5,12 +5,17 @@
 
 set -e
 echo ""
-echo "*** Running wgrib2 test"
+echo "*** Running wgrib2 tests"
 
-# Just run executable. For some reason it returns 8.
+# Just run executable; it returns 8.
+echo "*** testing run with no args..."
 ls -l ../wgrib2
 ../wgrib2/wgrib2 > out.txt  && exit 1
 [ $? -ne 8 ] && exit 2
 
+# Run an inventory on a test file. This does not work.
+#echo "*** testing inventory of gdaswave.t00z.wcoast.0p16.f000.grib2..."
+#./wgrib2 -inv ../tests/data/gdaswave.t00z.wcoast.0p16.f000.grib2 
```

Comment:
@webisu how should this wgrib2 command work, so that I get an inventory of the data file?

Reply:
OK, it's working now.